crates/biome_css_analyze/tests/specs/nursery/noUnknownUnit/invalid.css

```diff
@@ -0,0 +1,29 @@
+a { font-size: 13pp; }
+a { margin: 13xpx; }
+a { font-size: .5remm; }
+a { font-size: 0.5remm; }
+a { color: rgb(255pix, 0, 51); }
+a { color: hsl(255pix, 0, 51); }
+a { color: rgba(255pix, 0, 51, 1); }
+a { color: hsla(255pix, 0, 51, 1); }
+a { margin: calc(13pix + 10px); }
+a { margin: calc(10pix*2); }
+a { margin: calc(2*10pix); }
+a { -webkit-transition-delay: 10pix; }
+a { margin: -webkit-calc(13pix + 10px); }
+a { margin: some-function(13pix + 10px); }
+root { --margin: 10pix; }
+root { --margin: 10px + 10pix; }
```

Comment:
This has been approved.
I brought this from `stylelint`, but I feel that a formula that does not use `calc()` is not correct to begin with.